packages/angular/cli/lib/config/schema.json

```diff
@@ -633,9 +633,48 @@
               "default": false
             },
             "sourceMap": {
-              "type": "boolean",
+              "type": [
```

Comment:
Schema bookkeeping is getting pretty annoying. We should find a way around this sometime.

Reply:
Yeah, it’s pretty hard even just to add a new property. You have to add t in multiple places